src/components/structures/RoomView.js

```diff
@@ -296,7 +360,7 @@ module.exports = React.createClass({
 
         if (ev.getSender() !== MatrixClientPeg.get().credentials.userId) {
             // update unread count when scrolled up
-            if (!this.state.searchResults && this.refs.messagePanel && this.refs.messagePanel.isAtBottom()) {
+            if (!this.state.searchResults && this.state.atBottom) {
```

Comment:
Where did `this.state.atBottom` come from? Why is it A Thing? What differentiates it from `this.refs.messagePanel.isAtBottom()`?